process/mock/shardedDataMock.go

```diff
@@ -0,0 +1,134 @@
+package mock
```

Comment:
do we really need this ?

Reply:
deleted unnecessary file (no longer needed)